### PR TITLE
Fat 42326/add navigation

### PR DESCRIPTION
**Pull request by mariia-kiliushina**

### CODE REVIEW

File: src/navigation/tabs/MainTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import {Alert, BackHandler} from 'react-native';
+
+export type HomeTabsParamList = {
+  Home: undefined;
+  Transaction: undefined;
+  Budget: undefined;
+  Profile: undefined;
+};
+
+export const Main = () => {
```
Comment (MaximZybo) [inline src/navigation/tabs/MainTabs.tsx:18]: Rename file to index.tsx

File: src/navigation/tabs/MainTabs.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  const Tab = createBottomTabNavigator<HomeTabsParamList>();
+  useEffect(() => {
```
Comment (MaximZybo) [inline src/navigation/tabs/MainTabs.tsx:20]: Move this to Home screen

**Review by MaximZybo — CHANGES_REQUESTED** [review]

File: src/navigation/tabs/MainTabs.tsx (hunk 2 above)
Comment (mariia-kiliushina) [reply src/navigation/tabs/MainTabs.tsx]: Added new screen 'Main' that renders only tabs navigation. Please check

File: src/navigation/stack/index.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+export const ScreenNavigation = () => {
+  const Stack = createNativeStackNavigator<RootStackParamList>();
+  return (
+    <Stack.Navigator
+      screenOptions={{
+        headerShown: true,
+      }}
+      initialRouteName="Main">
+      <Stack.Screen name="Main" component={MainScreen} />
```
Comment (MaximZybo) [inline src/navigation/stack/index.tsx:19]: Add Tabs here, remove Main screen

**Review by MaximZybo — CHANGES_REQUESTED** [review]

File: src/navigation/tabs/MainTabs.tsx (hunk 2 above)
Comment (mariia-kiliushina) [reply src/navigation/tabs/MainTabs.tsx]: Have just deleted additional screen

**Pull request merged**